Classes/Custom/DataHandler.php

```diff
@@ -67,8 +67,10 @@ public function processDatamap_postProcessFieldArray( $status, $table, $id, arra
 		}
 
 
+		$prefix = 'aimeos_';
+
 		// Only listen to own plugin data
-		if( !empty( $listType ) && compare( $listType, 'aimeos_', 1 ) === 0 )
+		if( !empty( $listType ) && substr_compare( $listType, $prefix, 0, strlen( $prefix ) ) === 0 )
```

Comment:
But found an other problem. Was a really bad example I have found! For the moment it does nothing, but you could use it to remove deprecated fields if the flexform xml file is delete. Should I delete the file or what do we do? The ext_localconf.php has to be cleaned to...